routing/CMakeLists.txt

```diff
@@ -99,13 +101,15 @@ set(
   routing_session.hpp
   routing_settings.cpp
   routing_settings.hpp
+  speed_camera.cpp
+  speed_camera.hpp
   segment.hpp
   segmented_route.cpp
   segmented_route.hpp
+  serialize_speed_camera.cpp
+  serialize_speed_camera.hpp
```

Comment:
s/serialize_speed_camera/speed_cameras_serdes.hpp/

And unify ser and des.
Alternatively, in an older scheme we used it would be speed_cameras_serialization.hpp.

Reply:
есть TODO насчет того, чтобы объединить ser + des, давай пока оставим как есть?

Reply:
помержил ser + des тут